patients/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .patient import Patient
+from .doctor import Doctor
+from .appointment import Appointment
+from .invoice import Invoice
```

Comment:
Add new line at the end of the file from next time.

Reply:
Did the the requested changes in this [commit](https://github.com/Curewell-Homeo-Clinic/admin-system/pull/57/commits/62667cb9a08a9f33b24cde80830882c7700c6bad)